scripts/zmtelemetry.pl.in

```diff
@@ -340,4 +356,5 @@ sub linuxDistro {
 return ($kernel, $distro, $version);
 }
 
-
+1;
+__END__
```

Comment:
The 1 gives a defined exit value.  The __END__ tells perl that no further processing should be done.  So you can put the documentation after this line, using perldoc, etc and it won't be executed.  This is the standard, preferred thing to put at the end of a perl script.